install-dependencies.sh

```diff
@@ -28,32 +30,47 @@ if [ ${UBUNTU_VER} = '14.04' ]; then
   sudo apt-add-repository -y ppa:imnmfotmal/libccd
 fi
 sudo apt-get update
+
 # Programs
-sudo apt-get install -y --no-install-recommends build-essential cmake doxygen   \
-g++ git ipython octave python-dev python-h5py python-numpy python-pip           \
-python-scipy python-setuptools python-wheel wget mlocate
+sudo apt-get install -y --no-install-recommends build-essential cmake doxygen \
+  g++ git octave python-dev python-setuptools wget mlocate
+if [ ${UBUNTU_VER} = '14.04' ] || [ ${UBUNTU_VER} = '16.04' ] || [ ${UBUNTU_VER} = '18.04' ]; then
+  sudo apt-get install -y --no-install-recommends ipython python-h5py python-numpy \
+    python-pip python-wheel python-scipy
+elif [ ${UBUNTU_VER} = '20.04' ]; then
+  sudo apt-get install -y --no-install-recommends python2 
+  curl https://bootstrap.pypa.io/get-pip.py --output get-pip.py
```

Comment:
On a fresh install of Ubuntu:20.04, curl is not installed so pip2 fails to install.